services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequest.java

```diff
@@ -132,31 +169,78 @@ public int hashCode() {
         private Builder() {
         }
 
+        /**
+         * Determines the matching conditions for this query request. See {@link QueryConditional} for examples
+         * and constraints.
+         *
+         * @param queryConditional the query conditions
+         * @return a builder of this type
+         */
         public Builder queryConditional(QueryConditional queryConditional) {
             this.queryConditional = queryConditional;
             return this;
         }
 
+        /**
+         * Results are sorted by sort key in ascending order if {@link #scanIndexForward} is true. If its false, the
+         * order is descending. The default value is true.
+         *
+         * @param scanIndexForward the sort order
+         * @return a builder of this type
+         */
         public Builder scanIndexForward(Boolean scanIndexForward) {
             this.scanIndexForward = scanIndexForward;
             return this;
         }
 
+        /**
+         * The primary key of the first item that this operation will evaluate. Use the value that was returned for
+         * {@link Page#lastEvaluatedKey()} in the previous operation.
```

Comment:
The way this is written makes it sound like you have to put something in this field and that could cause confusion.

Reply:
Re-worded.